duckduckgo/duckduckgo-privacy-extension
# Collapse duplicate request rows in devtools panel

**Reviewer:** @jonathanKingston 



## Description:


When using the devpanel, there would often be duplicate requests which fill up the panel. This PR changes the behaviour to instead increment a counter next to the original request whenever duplicates are generated adjacent to one another.

## Steps to test this PR:


1. Open privacy-test-pages.glitch.me
2. Open the devtools panel, set the target to be the previously opened page
3. Open http://privacy-test-pages.glitch.me/tracker-reporting/1major-via-script.html
4. Request should be observed in dev panel
5. Action column should show `🚫 block (default block)`
6. Click "Home" (http://privacy-test-pages.glitch.me/index.html) then navigate back to http://privacy-test-pages.glitch.me/tracker-reporting/1major-via-script.html
7. Extra request should **not** be observed in devpanel
8. Action column should show `🚫 block (default block) [2]`
9. Navigate to http://privacy-test-pages.glitch.me/tracker-reporting/1major-via-img.html
10. Separate request should be observed as a new row

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
